App.tsx

```diff
@@ -59,11 +60,20 @@ const Section: React.FC<{
 
 const App = () => {
   const isDarkMode = useColorScheme() === 'dark';
+  const [isLoading, setIsLoading] = useState<boolean>(true);
 
   const backgroundStyle = {
     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
```

Comment:
Zijn deze donkere en lichtere kleuren getest op contrast?

Reply:
Oh dit is nog standaard react-native, hier heb ik niks van gemaakt. Wel handig om deze functionaliteit even te laten staan als we ook zelf de dark mode willen maken.

Reply:
Wellicht dan uitcommenten als dat kan? Dan weet je zeker dat bij het testen alles gaat zoals wij willen dat het gaat.

Reply:
Check